tools/schema_loader.cc

```diff
@@ -318,6 +323,217 @@ std::vector<schema_ptr> do_load_schemas(std::string_view schema_str) {
     return schemas;
 }
 
+struct sstable_manager_service {
+    db::nop_large_data_handler large_data_handler;
+    db::config dbcfg;
+    gms::feature_service feature_service;
+    cache_tracker tracker;
+    sstables::directory_semaphore dir_sem;
+    sstables::sstables_manager sst_man;
+
+    explicit sstable_manager_service()
+        : feature_service(gms::feature_config_from_db_config(dbcfg))
+        , dir_sem(1)
+        , sst_man(large_data_handler, dbcfg, feature_service, tracker, memory::stats().total_memory(), dir_sem) {
+    }
+
+    future<> stop() {
+        return sst_man.close();
+    }
+};
+
+mutation_opt read_schema_table_mutation(sharded<sstable_manager_service>& sst_man, std::filesystem::path schema_table_data_path,
```

Comment:
A thought (but feel free to reject): This function more-or-less reimplements a specific kind of query, I wonder if we can't give this and other Scylla "tools" the ability to do full-fleged CQL queries instead of implementing queries like this. The risk of this "reimplementation" is that first of all it might have a bug (even though I'm sure you tested it thoroughly) and second, that it might break in the future as our code changes, and nobody will notice.